src/riak_cs_gc_manager.erl

```diff
@@ -0,0 +1,285 @@
+%% ---------------------------------------------------------------------
+%%
+%% Copyright (c) 2007-2015 Basho Technologies, Inc.  All Rights Reserved.
+%%
+%% This file is provided to you under the Apache License,
+%% Version 2.0 (the "License"); you may not use this file
+%% except in compliance with the License.  You may obtain
+%% a copy of the License at
+%%
+%%   http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing,
+%% software distributed under the License is distributed on an
+%% "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+%% KIND, either express or implied.  See the License for the
+%% specific language governing permissions and limitations
+%% under the License.
+%%
+%% ---------------------------------------------------------------------
+%%
+%% State Diagram
+%%
+%% start -> idle -(start)-> running
+%%            ^               |
+%%            +----(finish)---+
+%%            +----(stop)-----+
+%%
+%% Message excange chart (not a sequence, but just a list)
+%%
+%%  Message\  sdr/rcver  gc_manager    gc_batch
+%%   spawn_link)                   --->
+%%   cancel)                       --->
```

Comment:
Start is used in the state diagram above, cancel is used here and command line interface.
Because stop is used in gen_(fsm|server) contexts as to stop process itself but it stops only child process, cancel is better wording?